src/Nethermind/Nethermind.Dsl.Test/DslRpcModuleTests.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Threading.Tasks;
+using Nethermind.Api;
+using Nethermind.Api.Extensions;
+using Nethermind.Blockchain.Processing;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Core.Extensions;
+using Nethermind.Core.Test.Builders;
+using Nethermind.Int256;
+using Nethermind.Serialization.Json;
+using NSubstitute;
+using NUnit.Framework;
+
+namespace Nethermind.Dsl.Test
+{
+    public class DslRpcModuleTests
+    {
+        private DslPlugin _dslPlugin;
+        private INethermindApi _api;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _dslPlugin = new DslPlugin();
+            _api = Substitute.For<INethermindApi>();
+            _dslRpcModule = new DslRpcModule();
+        }
+        
+        [Test]
+        public async Task undefined_watch_test(String script)
+        {
+           _dslRpcModule.dsl_addScript("WATCH House WHERE Author IS 0x22eA9f6b28DB76A7162054c05ed812dEb2f519Cd PUBLISH WebSockets ytext");
```

Comment:
the test name does not tell me what we want to check here, do we want to test whether there will be any errors or if an interpreter will be added to the pool with unsupported WATCH value?